bikeapp/models.py

```diff
@@ -0,0 +1,46 @@
+from datetime import datetime
```

Comment:
Very simplistic database structure. I would've liked to see a bit more complexity here too. :)